### PR TITLE
[coreanimation] Update for iOS 10 beta 1

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by VincentDondain** [comment]
`Introspection-mac` tests fail:

```
***** ApiFieldTest.FieldExists
[FAIL] Could not find the field 'kCAEmitterBehaviorSimpleAttractor' in /System/Library/Frameworks/QuartzCore.framework/QuartzCore
***** ApiFieldTest.NonNullNSStringFields
[FAIL] CoreAnimation.CAEmitterBehavior.SimpleAttractor
```

**Comment by xamarin-release-manager** [comment]
Build success

File: src/coreanimation.cs [hunk 1]
```diff
@@ -820,14 +850,18 @@ public interface CATextLayer {
 
 	[BaseType (typeof (NSObject))]
 	[Model]
-	[Protocol (IsInformal = true)]
+	[Protocol] // not information as of iOS 10+
```
Comment (VincentDondain) [inline src/coreanimation.cs]: Didn't you mean "not informal"?

File: src/coreanimation.cs [hunk 2]
```diff
@@ -163,14 +163,39 @@ public interface CADisplayLink {
 		[Export ("paused")]
 		bool Paused { [Bind ("isPaused")] get; set; }
 	
+		[Deprecated (PlatformName.iOS, 10,0, message: "use preferredFrameRate")]
```
Comment (VincentDondain) [inline src/coreanimation.cs]: `PreferredFrameRate` maybe as the C# versions is PascalCased?

File: src/coreanimation.cs (hunk 1 above)
Comment (spouliot) [inline src/coreanimation.cs]: yup

**Comment by VincentDondain** [comment]
LGTM 👍

**Comment by xamarin-release-manager** [comment]
Build success

**Pull request merged**